src/simulation_model/test_reference_simtel_configuration.py

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/python3
+r"""
+    Test reference sim_telarray configurations against configurations generated sim_telarray.
+
+    Reference configuration files are in `tests/resources/sim_telarray_configurations/` and are
+    used in the simtools configuration tests to compare simtools-derived model parameters /
+    configuration files using the simulation model database with those obtained using sim_telarray.
+
+    These tests compare files with model version 5.0.0 (prod5) and 6.0.0 (prod6) configuration
+    generated by sim_telarray (no later models are part of the sim_telarray distribution).
+
+    This tools should be executed in the simtools-dev container environment, as it modifies the
+    'cfg' directory in the sim_telarray installation:
+
+    1. start a 'simtools-dev' container (as outlined in simtools/docker/README.md)
+    2. download the reference configuration file from Zenodo using the script
+    `download_configuration_from_zenodo.py`
+    2. run this script:
+
+    ```
+    python ../simtools-extra/src/simulation_model/test_reference_simtel_configuration.py prod5-south
+    ```
+
+    Note: tool gives incorrect answers for prod(5,6)-north CT3 and CT4.
```

Comment:
Is this still true?

Reply:
Yes, remember this is due to the multiple includes / if statements in Konrad's configuration files. I really don't want to add a lot of complicated code to modify them while it is straight forward to compare those 3-4 parameters which are reported by hand.

Reply:
OK!